docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -0,0 +1,486 @@
+{
+  "$id": "https://person-api.sso.mozilla.com/profile/v2/schema",
+  "type": "object",
```

Comment:
Often you'll want to specify `"additionalProperties": false,` when you have type object, as it defaults to allowing unspecified properties.

Reply:
i thought of basically releasing a new profile schema when that's needed - in particular, we sign entire profiles and publishers of data have to be on the same profile version regardless, i.e. not sure how much it gives us to allow additional properties. That said if you've a use case or if i'm misunderstood this please correct me

Reply:
I meant you should disallow additional properties... JSON schema defaults to allowing them..

Reply:
Not a big thing... Since you're probably not validating against this schema... But if you are... Perhaps you have tests, disallowing additional properties ensure that they don't sneak in :)

Reply:
technically, we would disallow (like on v1) though I found this:
https://github.com/json-schema-org/json-schema-spec/blob/master/jsonschema-hyperschema.xml#L798-L800
So i haven't done it. I suspect it's not a huge issue either way though yeah, we could still use it to "make sure".